<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetLoginClass</class>
 <widget class="QWidget" name="widgetLoginClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>491</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>widgetLogin</string>
  </property>
  <widget class="QPushButton" name="m_loginButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #4a90e2;
color: white;
border: none;
border-radius: 5px;
padding: 12px;
font-size: 14px;
font-weight: bold;
}

QPushButton:hover {
background-color: #357abD;
}

QPushButton:pressed {
background-color: #2a5a8D;
}</string>
   </property>
   <property name="text">
    <string>Sign In</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Access Your Wallet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign in to access your digital wallet</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>136</y>
     <width>405</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_usernameEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>405</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
padding: 10px;
border: 1px solid #dddddd;
border-radius: 5px;
font-size: 14px;
}

QLineEdit:focus {
border: 1px solid #4a90e2;
}</string>
   </property>
   <property name="placeholderText">
    <string>Enter your username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_passwordEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>274</y>
     <width>405</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
padding: 10px;
border: 1px solid #dddddd;
border-radius: 5px;
font-size: 14px;
}

QLineEdit:focus {
border: 1px solid #4a90e2;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter your password</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>405</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="registerLink">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>405</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color: #4a90e2;
text-decoration: underline;
font-size: 14px;
}

QLabel:hover {
color: #357abD;
}</string>
   </property>
   <property name="text">
    <string>&lt;a href=&quot;#&quot;&gt;Don't have an account? Register&lt;/a&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
